dictation/debugging.py

```diff
@@ -1,8 +1,12 @@
 import time
 import traceback
 
-from talon import Module, actions, cron, noise, ui
-from talon.mac.ui import Element
+from talon import Module, actions, app, cron, noise, ui
+
+if app.platform == "mac":
+    from talon.mac.ui import Element
+else:
+    Element = type(None)
```

Comment:
I think it would be a bit more pythonic just try to import it, and then put the `Element = type(None)` in a `except ImportError` block.

Reply:
But, cool idea!

Reply:
I was waffling on that, so that works! (Devil's advocate says "don't want to suppress breakage if talon.mac.ui changes on the Mac", but being intention-revealing likely trumps that.)